int32 serialID

int32 driveDirection

int32 stop

int32 packIn
int32 packOut
int32 pause
